code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -17,3 +17,9 @@
 
 /obj/item/weapon/storage/briefcase/New()
 	..()
+
+/obj/item/weapon/storage/briefcase/MouseDrop_T(obj/item/weapon/restraints/handcuffs/I, mob/user)

```

Comment:
You really didn't test this very well 😛. There's no sanity checks for the first argument anywhere along this entire line of procs. You can attach _anything_ to briefcases to act as cuffs and subsequently qdel it. Including mobs.
![https://i.imgur.com/Cyq8ECN.png](https://i.imgur.com/Cyq8ECN.png)

Reply:
...lol.